grafast/grafserv/examples/example-lambda.mjs

```diff
@@ -0,0 +1,10 @@
+import { grafserv } from "grafserv/lambda";
```

Comment:
Normally we version our adaptors; is it likely we would need this with lambda? In general I'd err on the side of adding a version (even `v1`) just to leave space for future changes.